packages/core/src/forms/control-label/control-label.element.scss

```diff
@@ -22,23 +23,23 @@
     $cds-token-global-font-x-height
   );
 
-  font-size: $cds-token-typography-secondary-font-size;
-  font-weight: $cds-token-typography-secondary-font-weight;
-  color: $cds-token-typography-secondary-color;
-  line-height: $cds-token-typography-secondary-line-height;
-  letter-spacing: $cds-token-typography-secondary-letter-spacing;
+  font-size: $cds-token-typography-secondary-font-size !important;
+  font-weight: $cds-token-typography-secondary-font-weight !important;
+  color: $cds-token-typography-secondary-color !important;
+  line-height: $cds-token-typography-secondary-line-height !important;
+  letter-spacing: $cds-token-typography-secondary-letter-spacing !important;
```

Comment:
When using core in a app that has a global CSS framework defined already global elements like label/input collide. The important ensure slotted elements are consistent.